build/NuSpecs/Umbraco.Cms.nuspec

```
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+  <metadata minClientVersion="4.1.0">
+    <id>Umbraco.Cms</id>
```

Looks like the `version` entry is required for the XML schema to validate, so let's add it back using the `$version$` placeholder:
```suggestion
    <id>Umbraco.Cms</id>
    <version>$version$</version>
```